packages/gatsby/src/redux/types.ts

```diff
@@ -47,6 +47,9 @@ export interface IGatsbyConfig {
     title?: string
     author?: string
     description?: string
+    sireUrl?: string
+    // siteMetadata is free form
+    [key: string]: unknown
```

Comment:
Fail

Reply:
I mean - in context of gatsby core it doesn't matter much, because core doesn't make use of those fields really

Reply:
But it might matter when we use those types for public `index.d.ts` declaration instead of hand written one like we do right now

Reply:
Yeah, doesn't matter at all

Just weird to see those

Reply:
I needed it here I think for tests in particular which were setting random thing (really random):
```
    setSiteConfig({
      someRandomThing: `hi people`,
      plugins: [],
    })
```

Reply:
That's wrong one - that one was failing - it was for this one:
```
    const action = setSiteConfig({
      siteMetadata: {
        hi: true,
      },
    })
```

Reply:
```
someRandomThing: `hi people`
```

This is _almost_ as good as `your house is on fire`